homework/shelkovich_ilya/circle/src/test/java/com/tutrit/java/quickstart/service/CircleServiceTest.java

```diff
@@ -0,0 +1,26 @@
+package com.tutrit.java.quickstart.service;
+
+import com.tutrit.java.quickstart.bean.Circle;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CircleServiceTest {
+
+    Circle circle;
+    CircleService circleService;
+
+    @Before
+    public void setUp(){
+        this.circle = new Circle(4);
```

Comment:
new Circle could be created right in the test

Reply:
improved this part

Reply:
Indeed, I don't need to create circle every time. thx!